README.md

```diff
@@ -207,7 +214,7 @@ the `profileCustomizations` field:
 | `devHighNodeUtilizationThresholds` | `string` | Sets thresholds for the [HighNodeUtilization](https://github.com/kubernetes-sigs/descheduler#highnodeutilization) strategy of the `CompactAndScale` profile in the following ratios: `Minimal` for 10%, `Modest` for 20%, `Moderate` for 30%. Currently provided as an experimental feature.|
 |`devActualUtilizationProfile`|`string`| Sets a profile that gets translated into a predefined prometheus query |
 | `devDeviationThresholds` | `string` | Have the thresholds be based on the average utilization. Thresholds signify the distance from the average node utilization. An AsymmetricDeviationThreshold will force all nodes below the average to be considered as underutilized to help rebalancing overutilized outliers. Supported settings: `Low`: 10%:10%, `Medium`: 20%:20%, `High`: 30%:30%, `AsymmetricLow`: 0%:10%, `AsymmetricMedium`: 0%:20%, `AsymmetricHigh`: 0%:30% |
-| `devEnableSoftTainter` | `bool` | Have the operator deploying the soft-tainter component to dynamic set/remove soft taints as a hint for the scheduler based on the same criteria used for the descheduling |
+| `KubevirtRelieveAndMigrateSoftTainter` | `string` | Control if the KubeVirtRelieveAndMigrate profile will trigger the deployment of the soft-tainter component. Enabled by default. Not relevat for other profiles. |
```

Comment:
Wouldn't this be easier to reason about  with a bool named `DisableKubevirtRelieveAnMigrateSoftTainter`. If not specified then it is `false` by default. I have the impression that this would simplify things here as no extra type would be needed.

Reply:
@ingvagabund what do you suggest?